plugins/action_revoke_permissions/plugin.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "action_revoke_permissions",
+  "code": "revokePermissions",
+  "name": "Revoke Permissions",
+  "description": "Revokes existing permission(s) from user/org -- only sets existing permissions to inactive, doesn't actually delete records",
+  "required_parameters": ["username", "permissionNames"],
```

Comment:
Maybe username doesn't need to be required, since can be fetched from applicationData

Reply:
Similarly for `grantPermission`. I'm only suggesting to update the docs and `plugin.json` description

Reply:
Yup. Done 4bd37270